ui/src/views/dashboard/UsageDashboard.vue

```diff
@@ -137,7 +137,7 @@ export default {
     this.$store.watch(
       (state, getters) => getters.project,
       (newValue, oldValue) => {
-        if (newValue && newValue.id) {
+        if (newValue && newValue !== oldValue) {
```

Comment:
@Pearl1594 why not compare ids itself? This condition might be as good as `if (newValue) {`

Reply:
there is no id for default view @shwstppr